wunderground_upload.yaml

```diff
@@ -215,6 +229,12 @@ variables:
   indoorhumidity: >
     {{ sensor_indoorhumidity if sensor_indoorhumidity == ''
         else (states(sensor_indoorhumidity) | float(default=0)) }}
+  solarradiation: >
+    {{ sensor_solarradiation if sensor_solarradiation == ''
+        else (states(sensor_solarradiation) | float(default=0)) / 126.7 }} 
```

Comment:
What is this multiply factor that is used here?
Can you provide me all the attributes for this sensor?
The needed `unit_of_measurement` is `W/m²` according to [the API documentation](https://support.weather.com/s/article/PWS-Upload-Protocol?language=en_US).

Reply:
Native sensor reports in lux, but as you state, WUnderground wants w/m2.  Conversion factor between the two is 126.7
https://ambientweather.com/faqs/question/view/id/1452/

Reply:
Clear. Although I would like to check for the `unit_of_measurement` before doing the calculation.